### PR TITLE
Add marquee scrolling when title or artist name is too long

### PR DESCRIPTION
**Pull request by Ustice**
When the song title or artist is too long to fit in the details popup, it scrolls like a marquee.

Note that the animation is smooth, and the jumpiness is simply the gif recording.

### Showcase
mobile header display bug fixed

### CODE REVIEW

**Comment by Ustice** [comment]
How long does it take for the extension in the chrome store to update?

File: src/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Media Keys by Sway.fm",
-  "version": "4.42",
+  "version": "4.51",
```
Comment (msfeldstein) [inline src/manifest.json]: in the future don't bump the version, let me collect all changes and do that at once

Comment (Ustice) [inline src/manifest.json]: Will do, sorry. Do you need me to remove this from the PR?

**Comment by msfeldstein** [comment]
that'd be good actually, otherwise it causes me merge conflicts (not at all
a big deal but might as well keep it clean)

**Comment by Ustice** [comment]
Done

**Pull request merged**